src/main/java/com/google/growpod/LoadTestData.java

```diff
@@ -0,0 +1,175 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.growpod;
+
+import com.google.cloud.datastore.Batch;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Key;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery;
+import com.google.growpod.data.ContainsPlant;
+import com.google.growpod.data.Garden;
+import com.google.growpod.data.HasMember;
+import com.google.growpod.data.Plant;
+import com.google.growpod.data.User;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Helper methods to load and clear test data from any object implementing the DatastoreOptions
+ * interface.
+ */
+public class LoadTestData {
```

Comment:
Nit: my preference for a helper function wrapper is to make most of it as static/pure as possible. Take in args in the individual functions instead of holding state (instance, datastore)